src/gtk/textctrl.cpp

```diff
@@ -911,9 +942,21 @@ bool wxTextCtrl::Create( wxWindow *parent,
 
 GtkEditable *wxTextCtrl::GetEditable() const
 {
-    wxCHECK_MSG( IsSingleLine(), nullptr, "shouldn't be called for multiline" );
+    if( IsSingleLine() )
+        return GTK_EDITABLE(m_text);
+    else
+        return nullptr;
```

Comment:
Why is this necessary? We still shouldn't be calling this for multiline controls, it doesn't make sense as they don't have any `GtkEditable`.

It looks like this is a leftover and should be simply reverted.